packages/web_pointer_interceptor/lib/src/web.dart

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore: avoid_web_libraries_in_flutter
+import 'dart:html' as html;
+
+import 'package:flutter/widgets.dart';
+
+import 'shim/dart_ui.dart' as ui;
+
+const String _viewType = '__webPointerInterceptorViewType__';
+const String _debug = 'debug__';
+
+String _getViewType({bool debug = false}) {
+  return debug ? _viewType + _debug : _viewType;
+}
+
+void _registerWrapper({bool debug = false}) {
+  final String viewType = _getViewType(debug: debug);
+
+  ui.platformViewRegistry.registerViewFactory(viewType, (int viewId) {
+    final html.Element wrapper = html.DivElement();
+    if (debug) {
+      wrapper.style.backgroundColor = 'rgba(255, 0, 0, .5)';
+    }
+    return wrapper;
+  });
+}
+
+/// The mobile implementation of the PointerInterceptor widget.
+/// A Widget that prevents clicks from being swallowed by HtmlViewElements.
+class PointerInterceptor extends StatelessWidget {
+  /// Creates a PointerInterceptor for the web.
+  /// If the underlying viewFactories are not registered yet, it registers them.
```

Comment:
What does `viewFactories` refer to? Sounds like an implementation detail. Would a Flutter developer understand what this is talking about?

nit: Separate the second paragraph from the first by one empty line (here and other dartdocs in this PR)

Reply:
I actually removed the mentions to viewFactories from this method, you're right that it doesn't matter to the programmer.